_entities.md

```diff
@@ -251,7 +251,7 @@ type         | The nature of the transaction. Possible values are `deposit`, `tr
 message      | A message or note provided by the user at the time the transaction was initiated, with the intent of communicating additional information and context about the nature/purpose of the transaction.
 denomination | The funds to be transferred, as originally requested. See [Denomination](#denomination).
 fees         | The fees that were applied to the transaction. See [Fees](#fees).
-status       | The current status of the transaction. Possible values are: `pending`, `waiting`, `cancelled` or `completed`.
+status       | The current status of the transaction. Possible values are: `pending`, `processing`, `waiting`, `cancelling`, `cancelled`, `failed` and `completed`.
```

Comment:
Not sure if we are using `cancelling` :thinking:

Reply:
I'm not familiar with of all the code paths that could expose the various statuses, so I thought it would be safer to list all the possibilities present in the code. I can omit `cancelling` from this list if you think it's more appropriate.

Reply:
[Updated](https://github.com/uphold/docs/compare/7b759f7570c0d04c7c1acc2839bb27eb17c6778e..43e00c9b8cc7dd361cff9e4c09e9ec51edb6acfe): removed `cancelling` from the list.